addOns/oast/src/main/java/org/zaproxy/addon/oast/services/interactsh/InteractshOptionsPanelTab.java

```diff
@@ -179,6 +213,19 @@ public void saveParam(OptionsParam options) {
         param.setPollingFrequency(getPollingFrequencySpinner().getValue());
     }
 
+    private void refreshPayloadButtonEnabledState() {
+        if (newPayloadButton == null) {
+            return;
+        }
+
+        if (StringUtils.equals(getServerUrl().getText(), originalServerUrl)
```

Comment:
- `Objects.equals`
- SonarSource might want this inlined in the call to `setEnabled`.